docs/cloud/satellites.md

```diff
@@ -1,18 +1,6 @@
 # Earthly Satellites
 
-This feature is part of the Earthly Satellites & Earthly CI paid plans.
-
-{% hint style='danger' %}
-##### Important
-
-This feature is currently in **Beta** stage
-
-* The feature may break or change significantly in future versions of Earthly.
-* Give us feedback on
-  * [Slack](https://earthly.dev/slack)
-  * [GitHub issues](https://github.com/earthly/cloud-issues/issues)
-  * [Emailing support](mailto:support+satellite@earthly.dev)
-{% endhint %}
+This feature is part of the [Earthly CI paid plan](https://earthly.dev/pricing).
```

Comment:
We still have the sat plan.

```suggestion
This feature is part of the [Earthly Satellites & Earthly CI paid plans](https://earthly.dev/pricing).
```

Reply:
Oh I totally didn't see the small disclaimer at the bottom of the pricing tiers 

<img width="292" alt="Screen Shot 2023-04-27 at 8 14 00 AM" src="https://user-images.githubusercontent.com/3247216/234858727-1e6cce6d-5a26-477a-8dc2-5330b025eba4.png">

Maybe it was just me glancing over it too quickly, but it might be a bit too hidden

Reply:
This will change in the future, to make it more obvious :-)